perf/reader.go

```diff
@@ -163,7 +165,9 @@ type ReaderOptions struct {
 	// The number of written bytes required in any per CPU buffer before
 	// Read will process data. Must be smaller than PerCPUBuffer.
 	// The default is to start processing as soon as data is available.
-	Watermark int
+	Watermark     int
+	// This perf ring buffer will be written from backward and over writable.
+	OverWritable bool
```

Comment:
Kernel lingo makes sense for us since we've read the context, less so for other users! I'm fine with keeping the current name, but let's capitalize as `Overwritable` then since it's a single word. See https://en.wiktionary.org/wiki/overwritable